Matrix.SynapseInterop.Worker.AppserviceSender/Dto/AppserviceDto.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Matrix.SynapseInterop.Database.WorkerModels;
+using Newtonsoft.Json;
+
+namespace Matrix.SynapseInterop.Worker.AppserviceSender.Dto
+{
+    internal class AppserviceDto
```

Comment:
What's a DTO?

Reply:
Data Transfer Object (technically used incorrectly here, but it serves as an easy place for binding JSON to structured data without worrying about DB access)